ipynb/diversity-analysis/1-Summary-Table.md

```diff
@@ -0,0 +1,167 @@
+
+The goal of this notebook is to provide demographic summaries for participants in the American Gut and associated projects. We look at metadata, and summarize the available information.
+
+The information generated here will be used for tables 1 of the American Gut paper.
+
+We'll start by importing the necessary libraries.
+
+
+```python
+import matplotlib
+matplotlib.use('Agg')
+
+import os
+
+import numpy as np
+import pandas as pd
+```
+
+Next, we're going to locate the mapping file and raw OTU table.
+
+
+```python
+processing_dir = os.path.abspath('../primary-processing/agp_processing/')
+map_fp = os.path.join(processing_dir, '01-raw/metadata.txt')
```

Comment:
It's probably cleaner. I'm not sure if its ofscicating.

Maybe it should be set up with a data directory and a results directory were it's assumed that the data directory is within the respository, and runs based not he absolute path of the repository (erroring if the data isn't there) and then a results directory which can be set based on the users needs.

If we do that, should agp_processing in primary-processing be added to a git ignore?